event_manager/Event query_target
time[] stamp